GateKeeper.cpp

```diff
@@ -383,6 +385,105 @@ namespace pEp {
         return enabled;
     }
 
+    bool GateKeeper::is_update_available() {
+        bool result = true;
+        NTSTATUS status = BCryptOpenAlgorithmProvider(&hAES, BCRYPT_AES_ALGORITHM, MS_PRIMITIVE_PROVIDER, 0);
+        assert(status == 0);
+        result = status == 0;

```

Comment:
why checking each time if result = status == 0; ?

Better to initialize result to false by default instead to true, then you don't need to update every time with:
result = status == 0;